zookeeper-server/src/main/java/org/apache/zookeeper/server/ZKDatabase.java

```diff
@@ -149,7 +157,26 @@ public ZKDatabase(FileTxnSnapLog snapLog) {
                 DEFAULT_COMMIT_LOG_COUNT);
             commitLogCount = DEFAULT_COMMIT_LOG_COUNT;
         }
-        LOG.info("{}={}", COMMIT_LOG_COUNT, commitLogCount);
+
+        try {
+            commitLogSize = Double.parseDouble(
+                System.getProperty(COMMIT_LOG_SIZE,
+                    Double.toString(DEFAULT_COMMIT_LOG_SIZE)));
+            if (commitLogSize < DEFAULT_COMMIT_LOG_SIZE) {
```

Comment:
Even if there is this check on the logCount setting, I would leave this out as it makes it difficult to set a lower value.

Reply:
+1